README.md

```diff
@@ -8,6 +8,29 @@ Ansible role for installing and configuring an
 [OpenVPN](https://openvpn.net) server.  This role also enables IPv4
 NAT and iptables persistence.
 
+Note that this role cannot perform every step necessary to set up NAT.
+Once an instance is started up, one must determine the NAT interface
+and add an iptables rule:
+
+```console
+iptables -t nat -A POSTROUTING -s "${client_network_cidr}" -o "$interface" -j MASQUERADE
+```
+
+Next, one must save the iptables rules so they become persistent.
+This entails the commands
```

Comment:
I guess it could go either way here, but to my eyes, there should be a `:` after `commands` for consistency with the block above.

Reply:
"We will no longer use the following letters:  A, B, and C." vs "We will no longer use the letters A or B."  The typesetting doesn't change the sentence structure, so I'd argue for no colons here.

Reply:
I have cut the Gordian Knot by making a cosmetic change to the previous section such that it no longer requires a colon.  See commit 9074f93.

Reply:
Good work Alexander. 👍